assets/app/scripts/services/logs/logUtils.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .factory('logUtils', [
+    function() {
+      return {
+        addLineNumbers: function(str, padding) {
```

Comment:
Maybe I'm missing it, but I don't see where `addLineNumbers` and `toList` are called.